runtime/doc/options.txt

```diff
@@ -3499,7 +3499,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	if you want to use Vim as a modeless editor.
 	These Insert mode commands will be useful:
 	- Use the cursor keys to move around.
-	- Use CTRL-O to execute one Normal mode command |i_CTRL-O|).  When
+	- Use CTRL-O to execute one Normal mode command |i_CTRL-O|.  When
 	  this is a mapping, it is executed as if 'insertmode' was off.
```

Comment:
There are a few chunks missing from the merge of e4a3bcf28d92.

- Added description of `c_CTRL-N`/`c_CTRL-P` to `'incsearch'`.
- Rewording of `'langnoremap'` advice in the help for `'keymap'`.  This was removed in b2ff86843 but the advice should still be there, IMO, since it provides guidance to the user about the interaction between `'keymap'` and `'langremap'`.